core/trie/trie.go

```diff
@@ -234,17 +248,9 @@ func (t *Trie) Put(key, value *felt.Felt) (*felt.Felt, error) {
 			// we have to deference the Value, since the Node can released back
 			// to the NodePool and be reused anytime
 			old = *sibling.node.Value // record old value to return to caller
-			if value.IsZero() {
```

Comment:
nit: reword the now-outdated comment `Replace if key already exist` a few lines up from here. Maybe `Delete` instead of `Replace`

Reply:
Done :+1: